tests/testthat/test-nodeCentricSteinerTreeProblem.R

```diff
@@ -196,9 +185,22 @@ test_that("Trying MWCS solver against the lymphoma.stp instance in the ACTMOD St
 
   skip("SteinLib drosophila test takes too long even using CPLEX - although it does pass!")
 
-  drosophila001_igraph <- readMWCSgraph('./testData/ACTMOD/drosophila001.stp')
+  #drosophila001_igraph <- readMWCSgraph('./testData/ACTMOD/drosophila001.stp')
 
   # Takes around five minutes with CPLEX
-  #drosophila001_MWCS <- solveNodeCentricSteinerTreeProblem(drosophila001_igraph, timeOut = 7200,verbosity = TRUE, solverChoice = "RCPLEX")
+  #drosophila001_MWCS <- nodeCentricSteinerTreeProblem$new(drosophila001_igraph, verbose = T)$findSingleSteinerSolution(100)
+})
+
+
+test_that("nodeCentricSteinerTreeProblem max iteration warning",{
+  
+  expect_warning(gene42_MSTP <- nodeCentricSteinerTreeProblem$new( as.undirected(gene42_igraph), 
+                                     verbose = FALSE,
```

Comment:
```suggestion
                                     verbose = FALSE, solverTimeLimit=60,
```

A single minute rather than five should suffice to trigger the error. Running for 3 iterations, that could be a saving of 12 minutes!

Reply:
Actually, this run really quickly. It is not testing the time limit. It's actually testing that a warning (solution might be disconnected) is triggered when we reached the maximum number of iteration.

Reply:
I also tried to make a test for the time limit by reducing the solverTimeLimit to 1 second but I gave up as the example datasets we have are resolved "too" quickly by the solvers.

Reply:
Yeah - I wouldn't worry about a test on time limits. An improvement in solver and they could be affected all over again.